sensord/main.cpp

```diff
@@ -78,7 +107,7 @@ static void signalUSR2(int param)
     Q_UNUSED(param);
 
     qCWarning(lcSensorFw) << "Flushing sensord state";
-    qCWarning(lcSensorFw) << QString("  Logging level: %1").arg(logLevel);
+    qCWarning(lcSensorFw) << QString("  Logging level: %1").arg(logLevel).toUtf8().data();
```

Comment:
This doesn't seem to make sense. Also the existing toLocal8Bit() below looks pointless, logging should generally get QStrings.

Reply:
yup, but for whatever reason the '-' version outputs:  "\"  Logging level: 1"\n" i.e. unwanted double quotes appear from somewhere, possibly because there are leading space or something... and getting rid of QString stops that from happening.

Reply:
Adding noquote() seems to do the trick